backend/backend/.env

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY= %fv6p3@%o^d#wrlwmc)+&)e)fg&ugf@=^_w775qb61%r@8oi=k
```

Comment:
This file should be in `backend/.env` not `backend/backend/.env`, once it is moved there -> it will not be checked into git